app/views/collections/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= simple_form_for @collection do |form| %>
+<%= simple_form_for @collection, data: { turbo: false } do |form| %>
```

Comment:
To fix rendering of errors.

Reply:
What was the problem that turbo was causing here?

Reply:
What kind of errors?

Reply:
Turbo frames aren't added to the page. Thus, when validation failed and a page displaying the errors was being returned, that page wasn't being displayed. Hence, the errors weren't rendered to the user.

This tells turbo to stand down.

Reply:
What do you mean by "Turbo frames aren't added to the page."? How would they not be?

Reply:
https://github.com/sul-dlss/was-registrar-app/pull/464